views/lib/components/files-storage/forms/form-name-edit.hbs

```diff
@@ -1,9 +1,10 @@
-<div class="form-group">
-    <label for="newNameInput">{{$t "files.label.chooseANewName"}}</label>
-    <input id="newNameInput" class="form-control" type="text" />
-    <input
-        id="fileRecordId"
-        class="form-control"
-        type="hidden"
-    />
-</div>
+<div class='form-group'>
+  <label for='newNameInput'>{{$t 'files.label.chooseANewName'}}</label>
+  <input
+    id='newNameInput'
+    data-testid='rename-file-input'
+    class='form-control'
+    type='text'
+  />
+  <input id='fileRecordId' class='form-control' type='hidden' />
+</div>
```

Comment:
Could you add a trailer space line for this file so that eslint doesn't complain about it?

Reply:
Done